tools/review_pull_request.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+# License AGPLv3 (http://www.gnu.org/licenses/agpl-3.0-standalone.html)
+
+import argparse
+from collections import Counter
+from datetime import datetime, timedelta
+
+from github_login import login
+
+from oca_projects import OCA_USERNAME, OCA_REPOSITORY_NAMES
+
+contributions_cache = None
+
+
+def get_contribution_counts():
+    global contributions_cache
+
+    if contributions_cache is not None:
+        return contributions_cache
+
+    github = login()
+    contributions_cache = Counter()
+
+    for repository_name in OCA_REPOSITORY_NAMES:
+        repository = github.repository(OCA_USERNAME, repository_name)
+        for user in repository.contributors():
+            contributions_cache[user.id] += user.contributions_count
+
+    return contributions_cache
+
+
+def tag_new_pull_requests(repository_name):
+    github = login()
+    repository = github.repository(OCA_USERNAME, repository_name)
+    contributions = get_contribution_counts()
+
+    for pull_request in repository.pull_requests():
+        if contributions[pull_request.user.id] < 3:
+            pull_request.issue().add_labels('new contributor')
+
+
+def tag_all_new_pull_requests():
+    for repository_name in OCA_REPOSITORY_NAMES:
+        tag_new_pull_requests(repository_name)
+
+
+def close_abandoned_pull_requests(repository_name):
+    github = login()
+    repository = github.repository(OCA_USERNAME, repository_name)
+
+    for pull_request in repository.pull_requests():
+        full_pull_request = repository.pull_request(pull_request.number)
+
+        now = datetime.now(pull_request.created_at.tzinfo)
+        pr_is_old = now - pull_request.created_at > timedelta(days=31 * 6)
+
+        if pr_is_old and full_pull_request.comments_count == 0:
```

Comment:
We can have PRs with comments, but also abandoned. I think a better metric that the number of comments is the date of the PR < 6 months if no comment, but the date of the last comment < 6 months in case that you have comments.

Reply:
I was just following the requirements as lain out in the original issue. You may well be right, but I don't really want to get into discussing what the best metrics are. It should be easy enough to change, and feel free to do so, if you want to.

Reply:
Yeah, thanks, but as you know the API, please do it you. I give you my approval on behalf the @OCA/board